models/product-tailoring/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
```

Comment:
This is the main index file of the package so it should be exporting all the models.

Reply:
I think we're still missing the export of the types of the nested models.